pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -2776,54 +2842,4 @@ public List<String> getCommittingSegments(String realtimeTableName) {
   private List<String> getCommittingSegments(String realtimeTableName, Collection<String> segmentsToCheck) {
     return getCommittingSegments(realtimeTableName, segmentsToCheck, _helixResourceManager::getSegmentZKMetadata);
   }
-
-  public static List<String> getCommittingSegments(String realtimeTableName,
```

Comment:
Several methods/comments are moved. What's the reason? Seems these methods fit more in their original position